include/swift/AST/DiagnosticsSema.def

```diff
@@ -4807,6 +4807,12 @@ ERROR(return_init_non_nil,none,
       "'nil' is the only return value permitted in an initializer",
       ())
 
+ERROR(implicit_throws_super_init,none,
+      "missing call to superclass's initializer; "
+      "'super.init' is a throwing initializer and requires either an explicit "
```

Comment:
Maybe worth clarifying that we are referring to `super.init()`, not just any `super.init`?

Reply:
Yeah, totally!